Assets/Scripts/Models/Furniture.cs

```diff
@@ -422,136 +422,136 @@ public void ReadXmlPrototype(XmlReader readerParent)
                 case "Description":
-                reader.Read();
-                description = reader.ReadContentAsString();
-                break;
+                    reader.Read();
+                    description = reader.ReadContentAsString();
+                    break;
                 case "MovementCost":
-                reader.Read();
-                MovementCost = reader.ReadContentAsFloat();
-                break;
-            case "PathfindingModifier":
-                reader.Read();
-                PathfindingModifier = reader.ReadContentAsFloat();
-                break;
-            case "PathfindingWeight":
-                reader.Read();
-                PathfindingWeight = reader.ReadContentAsFloat();
-                break;
-            case "Width":
-                reader.Read();
-                Width = reader.ReadContentAsInt();
-                break;
+                    reader.Read();
+                    MovementCost = reader.ReadContentAsFloat();
+                    break;
+                case "PathfindingModifier":
+                    reader.Read();
+                    PathfindingModifier = reader.ReadContentAsFloat();
+                    break;
+                case "PathfindingWeight":
+                    reader.Read();
+                    PathfindingWeight = reader.ReadContentAsFloat();
+                    break;
+                case "Width":
+                    reader.Read();
+                    Width = reader.ReadContentAsInt();
+                    break;
                 case "Height":
-                reader.Read();
-                Height = reader.ReadContentAsInt();
-                break;
+                    reader.Read();
+                    Height = reader.ReadContentAsInt();
+                    break;
                 case "LinksToNeighbours":
-                reader.Read();
-                LinksToNeighbour = reader.ReadContentAsBoolean();
-                break;
+                    reader.Read();
+                    LinksToNeighbour = reader.ReadContentAsBoolean();
+                    break;
                 case "EnclosesRooms":
-                reader.Read();
-                RoomEnclosure = reader.ReadContentAsBoolean();
-                break;
+                    reader.Read();
+                    RoomEnclosure = reader.ReadContentAsBoolean();
+                    break;
                 case "CanReplaceFurniture":
-                replaceableFurniture.Add(reader.GetAttribute("typeTag").ToString());
-                break;
+                    replaceableFurniture.Add(reader.GetAttribute("typeTag").ToString());
+                    break;
                 case "DragType":
-                reader.Read();
-                DragType = reader.ReadContentAsString();
-                break;
+                    reader.Read();
+                    DragType = reader.ReadContentAsString();
+                    break;
                 case "BuildingJob":
```

Comment:
We should make a function for this case and reduce a bit the length of this function.